### DISCUSSION TITLE
自分か参加しているアドベントカレンダーに新規投稿があった時の通知をON/OFFできるようにしてほしい

### DISCUSSION THREAD

**Discussion by ysmb-wtsg** [increments/qiita-discussions#942] [ヘルプ] [answered]
通知が頻繁に飛んできてOFFにしたいと思ったが通知設定に項目がなく困っています

**Comment by tanaka-asami415** [comment]
@ysmb-wtsg 
リクエストありがとうございます！

ご不便をおかけして申し訳ありません。より良い形に改善していきたいため、もう少し詳しく教えていただけると嬉しいです🙏

リクエストいただいた「通知」は、Qiita上のベルマークの通知という理解で間違いないでしょうか？
![image](https://github.com/user-attachments/assets/e2180f96-25af-4b5a-9e7f-b7e9eaf8995d)

その前提で、質問させていただくのですが、アドベントカレンダーについて、「受け取りたい通知」と「不要な通知」がある状態でしょうか？

例えば…
- 「[購読しているカレンダー](https://help.qiita.com/ja/articles/qiita-adcal-view#part-f142300184c01639)」については、新規投稿時の通知を受け取りたいが、「自分が参加しているカレンダー」の通知はいらない
- 「[リマインダー](https://help.qiita.com/ja/articles/qiita-adcal-join#part-4f08284093b219b1)」については通知が欲しいが、その他はいらない
- [通知一覧](https://qiita.com/notifications?scope=advent_calendar) がもっと○○になっていたら、通知は今のままでもいい

など、お手すきの際に教えていただけると幸いです🙇

**Comment by ysmb-wtsg** [comment]
@tanaka-asami415 
>リクエストいただいた「通知」は、Qiita上のベルマークの通知という理解で間違いないでしょうか？

はい、認識相違ないです！

>アドベントカレンダーについて、「受け取りたい通知」と「不要な通知」がある状態でしょうか？

いえ、正確には **アドベントカレンダーの他人の投稿通知を全てOFFにしたい** というのが私の要望になります。

現状、以下のように、参加したアドベントカレンダーの投稿通知が勝手にきてしまいます。
![image](https://github.com/user-attachments/assets/e4398cbd-7de4-4291-8b5c-285ee0f993d6)

そして設定から通知をOFFにできないかと通知設定を一瞥したのですが、項目が見当たらずOFFにできず困っているといった状況です。

以上、ご確認よろしくお願いいたします。

**Reply by tomoasleep** [reply]
詳細を教えていただきありがとうございます！

ご不便をおかけして大変申し訳ありません。

確認したところ現時点では、参加しているアドベントカレンダーの通知に関しては、止める方法がない仕様となっておりました…🙇 
来年以降は、通知をオフにする設定を行えるように改善したいと思います。

**Reply by tsubolabo** [reply]
昨年の暮れに#598を要望しましたが、これと類似の問題のように思われます。
そのときには、

>無料で閲覧が可能なのに「購読」という言葉を使うと混乱を招く。
>>「有料でない」ことは伝わるようにしたいと思います

>参加記事の公開通知が「通知一覧」へ入るのは困る。
>>来年以降アップデート予定です。

という回答を頂いておりましたが、どちらも実現できていないようです。

**Reply by tanaka-asami415** [reply]
@tsubolabo

https://github.com/increments/qiita-discussions/discussions/598 へご報告ができておらず大変失礼しました。

>無料で閲覧が可能なのに「購読」という言葉を使うと混乱を招く。
>>「有料でない」ことは伝わるようにしたいと思います

上記については、ユーザーヒアリング等で複数名に確認した際、「有料と間違える」というご意見がなかったため、他の開発の優先度などから修正を見送る事となりました🙏

>参加記事の公開通知が「通知一覧」へ入るのは困る。
>>来年以降アップデート予定です。

こちらについては、いただいたご要望を「アドベントカレンダーの通知を分けて確認したい」という事だと捉えていたため、アップデート予定である旨を回答しておりました…🙇
通知をカテゴリーごとに確認できる機能はリリースが完了しています。
https://qiita.com/release-notes/2024-09-11

改めて、アドベントカレンダーの通知をオフにする設定を行えるように改善したいと思います。

**Reply by tsubolabo** [reply]
>上記については、ユーザーヒアリング等で複数名に確認した際、「有料と間違える」というご意見がなかったため、他の開発の優先度などから修正を見送る事となりました🙏

無料のものに「購読」は明らかに誤用です。数名への確認だけで「問題なし」と押し切って良い話ではありません。Qiitaの常識が疑われます。

**Reply by shiracamus** [reply]
「購読」は「subscribe」の和訳ですよね。購読＝購入して読む＝有料 というイメージがあるのでしょうが、無料（0円）のsubscribeはいくらでもありますよね。無料メルマガのsubscribeとか、このWebページの右側にも Nortifications の Subscribe/Unsubscribe ボタンがあります。
変更案としてはタグと同様に「フォローする」がいいかもしれませんね。

**Reply by tsubolabo** [reply]
@shiracamus 

貴重なご意見をありがとうございます。

>無料（0円）のsubscribeはいくらでもありますよね。

しかし、そうだからと言って、Qiitaで使っても問題ないという気持ちには、なかなかなりません。ネットを検索すれば「購読」という言葉に対する疑問の意見がいくらでも出てきます。以前の要望でも書いたように、「IT業界を主導しこれらを是正すべき立場にあるQiitaが、自らのページの中で使うべき言葉ではないと思います」という期待は今でも変わりません。

>変更案としてはタグと同様に「フォローする」がいいかもしれませんね。

私も同感です。

**Reply by tanaka-asami415** [reply]
ご意見ありがとうございます。

他の開発優先度と比較して一旦見送りとなってしまいましたが、引き続き改善を検討していきたいと思います🙇

**Comment by P-SiZK** [comment, marked as answer]
@ysmb-wtsg
大変お待たせいたしました :bow:

昨年ご報告いただいた通知設定の問題についての改善を行いました！
参加中カレンダーのフォローを外す、もしくは通知設定の「フォロー中のアドベントカレンダー」の設定を変更することで停止できるようになりました。

アドベントカレンダー関連の通知設定の詳細については、下記のリリースノートをご覧ください。
https://qiita.com/release-notes/2025-10-15

---

@tsubolabo @shiracamus
https://github.com/increments/qiita-discussions/discussions/598#discussioncomment-14695152 の返信と重複してしまいますが、
上記の改善に加えて、カレンダーの「購読」という表現をフォローに変更いたしました。

引き続き Qiita をよろしくお願いいたします。
